templates/partials/event-list.php

```diff
@@ -89,7 +89,7 @@ class="button is-small"
 				<?php else : ?>
 					<a href="<?php echo esc_url( Urls::event_trash( $event->id() ) ); ?>"
 						class="event-list-item-button is-destructive"
-						title="<?php echo esc_attr__( 'Delete', 'gp-translation-events' ); ?>">
+						title="<?php echo esc_attr__( 'Trash', 'gp-translation-events' ); ?>">
```

Comment:
Maybe we could use 'Move to trash' instead, I think @akirk wasn't a fan of using Trash as verb.

Reply:
I like "Move to trash"!